src/web/middleware/sentry.rs

```diff
@@ -158,6 +163,12 @@ where
                             trace!("Sentry: Not reporting error: {:?}", apie);
                             return future::ok(sresp);
                         }
+                        if let Some(metrics) = metrics {
```

Comment:
I think we want this in the above `!is_reportable()` block: reportable means it goes to sentry. If it's not reportable, we emit a metric instead (maybe "reportable" is not the best phrasing?).

Here's the [original from DbMiddleware](https://github.com/mozilla-services/syncstorage-rs/blob/0.9.0/src/web/middleware/db.rs#L178).

Reply:
Ah, right! thanks.